include/rqt_image_view/image_view.h

```diff
@@ -124,6 +129,8 @@ protected slots:
   bool pub_topic_custom_;
 
   QAction* hide_toolbar_action_;
+
+  int num_gridlines_;
```

Comment:
Please initialize the member variable in the constructor to `0` (or `1`?).

I am not sure what the exact semantic of this variable is. Based on the name I would expect `0` to not show any lines, `1` to show a single line in the middle of the image (horizontal and vertical).

Later on the code uses a default of `4`. Four lines would actually mean that non of the lines is in the middle. So my expectation is probably wrong. Can you please clarify the current semantic of this value.

Reply:
OK, thanks. I took all of your suggestions. Here's a new image: https://ibb.co/dYLw8w